.github/workflows/ci.yml

```diff
@@ -144,10 +144,17 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - uses: yezz123/setup-uv@v4
```

Comment:
Unsolicited fly-by: have you considered https://github.com/hynek/setup-cached-uv? Not that I've used either, yet 😆 .

Reply:
Sure, even https://github.com/hynek/setup-cached-uv/issues/5

The problem is the caching is slower than PyPI unless you are building a package.